dashboard/of-cloud-dashboard/handler.js

```diff
@@ -76,6 +76,18 @@ module.exports = async (event, context) => {
     headers['Content-Type'] = 'application/json';
   } else if (/.*\.map/.test(path)) {
     headers['Content-Type'] = 'application/octet-stream';
+  } else if (/^\/dist\/pub-cert.pem\/?$/.test(path)) {
```

Comment:
I think it might be useful to serve the key from the dashboard. Perhaps we can export the key during the setup and have it attached as as secret to the dashboard container?

Reply:
`/api/public-key.pem` <- it could then serve this value